### ISSUE TITLE
Commercialism Policy

### ISSUE THREAD

**Issue by please-wait** [Finhaven/blockchainYVR#5]
This started out for me as a hassle in a peer review process, but I am increasingly starting to admire [ASHRAE's Commercialism Policy](https://www.ashrae.org/File%20Library/Conferences/Speaker%20Resources/Commercialism-Policy-November-2016.pdf). For example, have a look at their [presentation template](https://www.ashrae.org/File%20Library/Conferences/Speaker%20Resources/2018-BPA_PPT_Template.pptx).

As per policy, they expect that:
- Any reference to an initiative or software is for clarity. 
- Software name is mentioned only once in the beginning, and afterwards referred to generically (Software 1, Software 2)
- Equipment branding is removed from the photos
- It is not suggested that ASHRAE Approves the product in any way
- Potential bias is listed
- Presenter's logo is only on the first slide

Specially the second item sounds awkward but interesting.

They can be so strict with all of it because they are big. But the underlying idea is: **a tool is a tool and nothing more, it can be substituted for something else, let's talk about what was achieved.** I think we should be using the same principle to the extent that it doesn't disincentivize presentation.

What would you think about having a commercialism note for presenters?

**Comment by bmann** [comment]
Yes, this sounds good!

Shall we create a "background for presenters" page which contains this and other info? I think that's a good next step.

(this is basically a no shilling rule)

I didn't make my PR yet, but I have a refactor of the site.

**Comment by please-wait** [comment]
Yes, a wiki/website page for that would be great! Ideally we can communicate all of it in one paragraph and a few bullets:

 - [ ] **Compatibility:** The presentation should...
 - [ ] **Transparency:** Bias and affiliation should be explained.
 - [ ] **Fairness:** What are the alternatives? what are the limitations? (encouraging comparison [compasses](https://www.smartinsights.com/wp-content/uploads/2018/01/Percentage-of-mobile-use-by-country-700x496.png) or [charts](http://orgweaver.com/wp-content/uploads/2017/04/Org-chart-software-alternative-comparison-2017.png)?)
 - [ ] **Commercialism:** Have the logo only on the first and last slide. Only mention a brand if that's necessary, put contact info in the last slide.
 - [ ] **Repeatability:** Due diligence in making claims. Code must be open for test and review. Put permalinks to the repo and the presentation file.

Am I missing anything?

**Comment by please-wait** [comment]
Alright, doacracy on this. Here is my proposal:

_____

BlockchainYVR hosts talks specifically for developers working on decentralization challenges. In each event, the presenters lift the layer of marketing from a cryptocurrency-related product and solely explore it as a tool. The presentations are often filled with terminal screens, code snips, buggy demos, and waiting for transaction confirmation. The length of each presentation can be from 5 to 20 minutes.

The presenters must maintain the following:

- **Compatibility:** The presentation content should be for the developer target audience.
- **Transparency:** You should briefly clarify your potential source of bias or any affiliation.
- **Fairness:** Dedicate an slide or more to introducing the alternatives. What are the limitations and advantages of your approach compared to those? A way of approaching this is by a comparison [compass](https://www.smartinsights.com/wp-content/uploads/2018/01/Percentage-of-mobile-use-by-country-700x496.png) or [chart](http://orgweaver.com/wp-content/uploads/2017/04/Org-chart-software-alternative-comparison-2017.png).
- **Commercialism:** If you want to include your company logo, have it only on the first and last slide. Only mention something if that's necessary for delivering your content.
- **Repeatability:** Do the due diligence in making claims. The code you present must be open for test and review. To facilitate that, prepare permalinks to your code repository and the slides before your presentation, to share.
